app/src/main/java/com/example/unlibrary/util/PushNotificationSender.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * PushNotificationSender
+ *
+ * November 22, 2020
+ *
+ * Copyright (c) Team 24, Fall2020, CMPUT301, University of Alberta
+ */
+
+package com.example.unlibrary.util;
+
+import android.util.Log;
+import android.view.View;
+
+import com.android.volley.toolbox.JsonObjectRequest;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Creates JSONObject and sends it to FCM
+ */
+public class PushNotificationSender {
+
+    private static final String TAG = PushNotificationSender.class.getSimpleName();
+    private static final String FCM_API = "https://fcm.googleapis.com/fcm/send";
+    private static final String SERVERKEY = "key=AAAAAkUm6fE:APA91bE_uyTQS3tH0vG_JG7qDTkwxqGvL9tmwlPo1mPhd8jwF714tTa_tJzq6Kg16MoqJotD3zAejCkvqN2xfjjaQ9qR_T4R6GxGctES6DNhlANWR7QtvDDMNfUzIys3OZK1SsNUzgSO";
+    private static final String CONTENTTYPE = "application/json";
```

Comment:
Nit use _ to separate words
```suggestion
    private static final String SERVER_KEY = "key=AAAAAkUm6fE:APA91bE_uyTQS3tH0vG_JG7qDTkwxqGvL9tmwlPo1mPhd8jwF714tTa_tJzq6Kg16MoqJotD3zAejCkvqN2xfjjaQ9qR_T4R6GxGctES6DNhlANWR7QtvDDMNfUzIys3OZK1SsNUzgSO";
    private static final String CONTENT_TYPE = "application/json";
```